### PR TITLE
fix: open active patient space on login and return to chats from elsewhere

### PR DESCRIPTION
**Pull request by suprko**
#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/8951/mc-stay-in-patient-space-when-switching-tabs
 
#### Testing instructions  
In MC, when (a) patient space is open with no active chat, or (b) a Patient Room or Internal Room is open:
* active patient space/room should show on login
* patient sidebar should show when navigating away from chats and then back again

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: src/whitelabel/helpers/elements.jsx [hunk 1]
```diff
@@ -212,6 +219,33 @@ class ELEMENTS {
...
+            goToActiveChat: () => {
+                routerStore.navigateTo(routerStore.ROUTES.chats);
+            }
+        };
+
+        if (config.whiteLabel.name === 'medcryptor') {
+            // In MC, need to check if activeChat is in patient space, and activate that space if so.
+            obj.goToActiveChat = () => {
+                if (chatStore.activeChat && chatStore.activeChat.isInSpace) {
+                    console.log('----------------------------------first');
```
Comment (anri-asaturov) [inline src/whitelabel/helpers/elements.jsx]: remove debug logs

File: src/whitelabel/helpers/elements.jsx [hunk 2]
```diff
@@ -212,6 +219,33 @@ class ELEMENTS {
 
         return obj;
     }
+
+    get loading() {
```
Comment (anri-asaturov) [inline src/whitelabel/helpers/elements.jsx]: why is this a getter? It's gonna execute every time you access it...

Comment (suprko) [reply src/whitelabel/helpers/elements.jsx]: The object doesn't return right when it's not a getter... Loading.jsx throws an error `ELEMENTS.loading.goToActiveChat() is not a function`

Comment (anri-asaturov) [reply src/whitelabel/helpers/elements.jsx]: Then you must be doing something in a wrong order. This is a bad design, but I don't have time to debug and offer fixes, so, based on the fact it's being called once in an app session I'll approve it.

**Comment by anri-asaturov** [comment]
Ready to test.

**Comment by suprko** [comment]
@ebilg done

**Comment by ebilg** [comment]
can be merged after release

**Pull request merged**